docs/howtos/customization/create_custom_service.md

```diff
@@ -138,3 +138,29 @@ For app charts the keys are simple strings for the `userConfig` map. They do not
 For service charts the keys are proper paths into the set of fields exposed by the chart through its
 `values.yaml` file. Nesting is expressed using dots (`.`). As example see `ingress.enabled` above.
 :::
+
+### Private repositories and registries
+
+To use charts stored in private repositories or OCI registries that requires authentication you can specify a secret where the credentials are stored.
```

Comment:
```suggestion
Specify a secret storing credentials when using charts provided by private repositories or OCI registries that require authentication.
```

@jhkrug Maybe you have something better ?

Reply:
Not sure about this one, let's wait for one more proposal from @jhkrug

Reply:
How about this. Two shorter, active voice, more simple, sentences.

You can use charts stored in private repositories or OCI registries. You will need to use a secret that provides credentials.

Reply:
Sounds good to me. @andreas-kupries wdyt?

Reply:
:+1:

Reply:
Thank you both, @andreas-kupries @jhkrug  done :+1: